buildtools/README.md

```diff
@@ -1,11 +1,56 @@
-# Setup
+# AWS.Deploy.Tools.CI
 
-1. Create CF template using `buildtools/ci.template`
-2. Copy output `CodeBuildProjectName` & `OidcRole` output variables.
-3. Create `CI_AWS_ROLE_ARN` repository secret with `OidcRole` value and
-   `CI_AWS_CODE_BUILD_PROJECT_NAME` repository secret with `CodeBuildProjectName`
-   value.
-4. Voila!
+The `AWS.Deploy.Tools.CI` project is used to setup the CI/CD PR check between the [AWS.Deploy.Tools](https://github.com/aws/aws-dotnet-deploy) GitHub Repo and the AWS CodeBuild project that will be executed by the GitHub workflow `AWS CodeBuild CI (codebuild-ci.yml)`.
+
+The workflow will be executed starting from `AWS CodeBuild CI (codebuild-ci.yml)` PR check, which will call a Lambda function in an `AWS Account` (which will act as a load balancer) which will ping 1 or many `Test Runner` accounts to check for availability.
+
+## Main Account (Load Balancer)
```

Comment:
As discussed with the team, we will move forward without implementing a more automated deployment approach.

Reply:
* Can you summarize the discussion here for future members of the team, those who weren't at the meeting, etc.
* @normj suggested that we should at least deploy the Lambda via the CLI instead of via Visual Studio which should allow us to write out more of the settings ahead of time.